assets/terraform-modules/packet/flatcar-linux/kubernetes/calico-host-protection.tf

```diff
@@ -1,3 +1,10 @@
+resource "local_file" "calico_crds" {
```

Comment:
Can we add a TODO comment here? Ideally pointing to an issue to fix this properly.

Reply:
I'm not sure to what issue I should point this to, so I'll put a link to the old issue for now.

I think conceptually it's correct that this chart gets installed in bootkube, as it handles the firewall, which we want to have as soon as possible (BTW, it still doesn't provide 100% protection, as bootstrap kube-apiserver will be exposed to the internet).

In the future, we may have a closed cluster from the internet until components operator kicks in and provisions those policies, then it can be moved from bootkube.